cluster-autoscaler/cloudprovider/aws/aws_wrapper.go

```diff
@@ -261,3 +149,58 @@ func (m *autoScalingWrapper) getAutoscalingGroupNamesByTags(kvs map[string]strin
 
 	return asgNames, nil
 }
+
+func (m *awsWrapper) getInstanceTypeByLaunchTemplate(launchTemplate *launchTemplate) (string, error) {
+	params := &ec2.DescribeLaunchTemplateVersionsInput{
+		LaunchTemplateName: aws.String(launchTemplate.name),
+		Versions:           []*string{aws.String(launchTemplate.version)},
+	}
+
+	describeData, err := m.DescribeLaunchTemplateVersions(params)
```

Comment:
Would be nice to export metric for that API call (`start := time.Now(); describeData, err := m.DescribeLaunchTemplateVersions(params); observeAWSRequest("DescribeLaunchTemplateVersions", err, start)`).  Those metrics were introduced after this PR was opened so most will be picked up with a rebase, but that function is new.

Reply:
Fixed.